### ISSUE TITLE
"Missing promise in ring" blows up Deno in 1.38.1 (but not 1.38.0)

### ISSUE THREAD

**Issue by zefhemel** [denoland/deno#21176]
I have very little to go on here, but since I upgrade to Deno 1.38.1 (from 1.38.0) my process sometimes crashes with:

```
error: Uncaught (in worker "") "Missing promise in ring @ 41720"
error: Uncaught (in promise) Error: Unhandled error in child worker.
    at Worker.#pollControl (ext:runtime/11_workers.js:159:19)
    at eventLoopTick (ext:core/01_core.js:189:11)
```

When I downgrade to 1.38.0 all seems fine.

Because it's random, I don't know if this is related to the worker aspect or something else. Does this ring a bell? I'm not even sure what "ring" is.

**Comment by zefhemel** [comment]
A potential hint may be that I'm using Deno's KV quite heavily (although not directly from a worker).

**Comment by 0f-0b** [comment]
Looks like this has already been fixed in denoland/deno_core#323.

**Comment by bartlomieju** [comment]
Thanks for the report; as pointed out it's been fixed in https://github.com/denoland/deno_core/pull/323 and will be released this week in Deno v1.38.2.

**Comment by zefhemel** [comment]
Awesome, thanks!
